Update admin-promotion instructions; clarify ID for evacuation (#2997)
matrix-org/dendrite: docs/administration/4_adminapi.md

in the URL. It may take some time to complete. A JSON body will be returned containing
the user IDs of all affected users.

> If the room has a published address set, the room's ID will not be in the URL, but may
> be found as the room's "internal ID" (e.g. under Advanced within the room settings in
> Element.)

Reviewer: Does this sound OK?
```suggestion
If the room has an alias set (e.g. is published), the room's ID will not be visible in the URL, but it can
be found as the room's "internal ID" in Element Web (Settings -> Advanced)
```

Author: Yes, that looks better, thank you!
